__tests__/workers/postUpdated.ts

```diff
@@ -100,6 +101,23 @@ const createDefaultKeywords = async () => {
   });
 };
 
+const DEFAULT_QUESTIONS = [
+  'What is the one thing you need to truly understand neural networks?',
+  'How do neural networks learn?',
+  'What is the process of training a neural network?',
+];
+
+const createDefaultQuestions = async (postId: string) => {
+  const repo = con.getRepository(PostQuestion);
+
+  for (const question of DEFAULT_QUESTIONS) {
+    await repo.insert({
+      postId,
+      question,
+    });
+  }
```

Comment:
No prio, but think this would also work:
```
await repo.save(DEFAULT_QUESTIONS.map((question) => ({ postId, question })));
```

Reply:
great call, thank you 🙏